redis.ts

```diff
@@ -330,6 +331,32 @@ export class RedisImpl implements Redis {
     return this.execStatusReply("CLIENT", "SETNAME", connectionName);
   }
 
+  clientTracking(opts: ClientTrackingOpts) {
+    const args: (number | string)[] = [opts.mode];
+    if (opts.redirect) {
+      args.push("REDIRECT", opts.redirect);
+    }
+    if (opts.prefixes) {
+      opts.prefixes.map((prefix) => {
```

Comment:
I think that `forEach` should be used instead of `map`. :slightly_smiling_face:
```suggestion
      opts.prefixes.forEach((prefix) => {
```

Reply:
Oh, haha, yes indeed. Thank you! Fixed.